.github/workflows/podman-nightly-e2e.yaml

```diff
@@ -112,7 +112,7 @@ jobs:
           quay.io/odockal/pde2e-podman:v0.0.1-snapshot  \
             pd-e2e/run.ps1 \
               -downloadUrl "https://api.cirrus-ci.com/v1/artifact/github/containers/podman/Artifacts/binary/podman-remote-release-windows_amd64.zip" \
-              -version '4.8.0-dev' \
+              -version '5.0.0-dev' \
```

Comment:
nightly are based on head not on the next minor release branch?

Reply:
@adrianriobo I guess yes: https://blog.podman.io/2023/11/podmans-main-branch-is-now-5-0-0-dev/